src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -59,7 +59,7 @@ def load_arguments(self, _):
     with self.argument_context('keyvault create') as c:
         c.argument('resource_group_name', resource_group_name_type, required=True, completer=None, validator=None)
         c.argument('vault_name', completer=None)
-        c.argument('sku', arg_type=get_enum_type(SkuName))
+        c.argument('sku', arg_type=get_enum_type(SkuName, default=SkuName.standard.value))
```

Comment:
Doesn't seem to be the purpose on this PR, so just double-check it was not added by mistake.

Reply:
It is deliberate. Before, SkuName was imported in the `custom.py` file and the default was applied that. When I had to move the SkuName import into the `load_arguments` method (to defer the azure.mgmt import) I had to switch to using this style of supplying the default (which is how the multi-API packages have to do it anyways).